app/src/main/java/uc/edu/klopfsea/groupscheduler/ui/main/AddEvent.kt

```diff
@@ -133,7 +133,10 @@ class AddEvent : Activity(), AdapterView.OnItemSelectedListener {
         if ("S" in days)
             dayString.append("S,")
         if ("U" in days)
-            dayString.append("U,")
+            dayString.append("U,")***/
+        for(i in days){
+            dayString.append("$i, ")
```

Comment:
This suggestion I am not sure about because I have not fully understand how this function executes therefore this change might not behave the way the function might have been intended.

Reply:
Have a look at the design doc to see the intended functionality.  Or, ask the group.